### ISSUE TITLE
Images in item section with full system path

### ISSUE THREAD

**Issue by RobertBozic** [ben-xo/dir2cast#16]
I have problems with images in item section of podcast:
Images are written with full filesystem path, instead of url:
```
<item> 
xxxxxx 
<itunes:image href="/var/www/users/show/www/show1/show_1.jpg"/> 
xxxxxx 
xxxxxx 
<pubDate>Tue, 06 Mar 2018 12:22:22 +0000</pubDate> 
xxxxx 
<image>/var/www/users/show/www/show1/show_1.jpg</image> 
</item>
```

Thanks, have a good day!

**Comment by ben-xo** [comment]
Is this a new issue since you upgraded versions, or is this the first time you've used it?

Try an older version, or try setting some of the options in the .ini, as this is generally what happens if you're looking at the files locally

**Comment by ben-xo** [comment]
I found the bug (whoops - a year and a half later). Sorry for ever doubting you. I'll push a fix shortly

**Comment by ben-xo** [comment]
Should be fixed in v1.18
